packages/cli/src/commands/accounts/add.ts

```diff
@@ -0,0 +1,46 @@
+import {Command} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import * as Heroku from '@heroku-cli/schema'
+import {add, list} from '../../lib/accounts/accounts'
+
+export default class Add extends Command {
+  static description = 'add a Heroku account to your cache'
+
+  static args = {
+    name: Args.string({description: 'name of Heroku account to add', required: true}),
+  }
+
+  static example = 'heroku accounts:add my-account'
+
+  async run() {
+    const {args} = await this.parse(Add)
+    const {name} = args
+    const logInMessage = 'You must be logged in to run this command.'
```

Comment:
I don't actually encounter this message when i'm not logged in:
<img width="362" alt="Screenshot 2025-03-20 at 1 44 54 PM" src="https://github.com/user-attachments/assets/aab33f88-2e6d-4c44-ac6b-f599b11402bf" />

Reply:
Thank you. I will check on this.

Reply:
This should be fixed now. Instead of an error message you should be prompted to log in.